Add some helper docs for contributing to the project (#83)
wearerequired/traduttore: .github/PULL_REQUEST_TEMPLATE.md

**Description**
<!-- Please describe what you have changed or added -->

**How has this been tested?**
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->

**Screenshots**

> If applicable, add screenshots that verify the behavior of your changes

Reviewer: ```suggestion
If applicable, add screenshots to show the visual change.
```

Author: FWIW I copied that sentence from the Gutenberg repo if you wanna suggest that change there as well :-)

Reviewer: You seem to have used an older version. https://github.com/WordPress/gutenberg/tree/master/.github
